Library/Formula/gdk-pixbuf.rb

```diff
@@ -52,12 +52,14 @@ def install
   def post_install
     # Change the version directory below with any future update
     ENV["GDK_PIXBUF_MODULEDIR"]="#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
+    ENV["GDK_PIXBUF_MODULE_FILE"]="#{lib}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
```

Comment:
I don't think it should be; I ran into problems when putting it there; performing `brew rm` would not clear out the non-symlink file in `#{HOMEBREW_PREFIX}/lib`, and so subsequent `brew install gdk-pixbuf` operations would fail.  With this setup, the file is placed within the formula's Cellar, and is then symlinked into the global namespace, which works out better for everyone, as far as I can tell.